apps/charterafrica/src/payload/fields/post.js

```diff
@@ -0,0 +1,65 @@
+import linkGroup from "./linkGroup";
+import richText from "./richText";
+import slug from "./slug";
+
+const setPublishedOn = ({ value }) => {
+  return new Date(value);
+};
+
+const postFields = [
+  {
+    name: "title",
+    label: {
+      en: "Title",
+      fr: "Titre",
+      pt: "Título",
+    },
+    type: "textarea",
+    localized: true,
+    required: true,
+  },
+  slug(),
+  {
+    name: "authors",
+    required: true,
+    type: "relationship",
+    relationTo: "author",
+    hasMany: true,
+  },
+  {
+    name: "image",
+    label: {
+      en: "Image",
+      pt: "Imagem",
+      fr: "Image",
+    },
+    type: "upload",
+    relationTo: "media",
+    required: true,
+  },
+  richText({
```

Comment:
We need the full `content` here in addition to excerpt/summary. The full content should probably be a block that allows various fields such as richText, media and video/ifram embed.

Reply:
@kilemensi won't video embed be the same as media upload?

Reply:
No @koechkevin ... video embeds usually use youtube or similar services. People shouldn't be uploading videos to the CMS.

Reply:
So we only give users a text field to enter embed URL. Right?

Reply:
Yeah, what does Ghost/WP give? Anyways, may be it should be a code field similar to the newsletter?

Reply:
We can give users to select the type of embed they'd like to use.

Either pasting custom url or embed code. The approach we used on links